app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :comments, foreign_key: :AuthorId
+  has_many :posts, foreign_key: :AuthorId
+  has_many :likes, foreign_key: :AuthorId
+
+  def check_last_three_posts
+    posts.limit(3).order(created_at: :desc)
+  end
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you update this method content to this `posts.order(created_at: :desc).limit(3).` instead `posts.limit(3).order(created_at: :desc)`. I think this would be good compared to the ones you used before